manifest.json

```
@@ -4825,6 +4825,29 @@
                 }
             }
         },
+        "net.catshark.rearmature": {
+            "name": "ReArmature",
+            "description": "Adds a button to SkinnedMeshRenderers that sets up its bones to your avatars armature. Even if there are new bones.",
+            "category": "Inspectors",
+            "website": "https://github.com/CatSharkShin/ReArmature",
+            "sourceLocation": "https://github.com/CatSharkShin/ReArmature",
+            "authors": {
+                "CatSharkShin": {
+                     "url": "https://github.com/CatSharkShin"
```

```suggestion
                    "url": "https://github.com/CatSharkShin"
```
Looks like there's an extra space here. This is the last issue preventing me from merging this.